src/showcase/stories/angular/sidebar/icon-sidebar-button/icon-sidebar-button.stories.ts

```diff
@@ -9,6 +9,19 @@ import {
 } from '@storybook/angular';
 import { InputType } from '@storybook/types';
 
+const type: InputType = {
```

Comment:
isn't that a little bit too much do display controls which will never be used? they are just inherites by our button base stuff

Reply:
if you check 
https://lyne-angular-storybook.app.sbb.ch/?path=/story/elements-sbb-sidebar-sbb-icon-sidebar-button--default
the type and button are available but as 'object' type, so in my opinion is better to have it with the right type

otherwise, we can remove the two controls from any component which inherits from button-base